### PR TITLE
Add support for postgres urls

### PR DESCRIPTION
**Pull request by hetoug**
Support for postgres urls as described in Issue #11

### CODE REVIEW

**Comment by sanpii** [comment]
Oops, I didn’t see your PR.

File: src/main.rs [hunk 1]
```diff
@@ -103,18 +103,27 @@ fn main() -> Result<()> {
...
-                config.password = Some(password.trim_matches('\n').to_string());
-
-                elephantry::Pool::from_config(&config)
-            } else {
-                Err(err)
+    if let Ok(client) = elephantry::Pool::new(&opt.dbname.clone().expect("No database name given"))
+    {
+        Ok(client)
+    } else {
+        {
```
Comment (sanpii) [inline src/main.rs]: There is an extra indent level, you can remove it.

Comment (hetoug) [reply src/main.rs]: Removed.

**Comment by sanpii** [comment]
Merged, thank you.

**Pull request closed without merging**